<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agregartarea</class>
 <widget class="QWidget" name="agregartarea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(125, 255, 251);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>AGREGAR TAREA</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>TIPO : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>NUMERO : </string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="btAlta">
           <property name="text">
            <string>ALTA</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3" colspan="2">
          <widget class="QPushButton" name="btnAgregar">
           <property name="text">
            <string>AGREGAR</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLineEdit" name="inNombreTarea"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>NOMBRE DE LA TAREA : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="inNumero"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>FECHA : </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>PRIORIDAD :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QRadioButton" name="btMedia">
           <property name="text">
            <string>MEDIA</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QRadioButton" name="btBaja">
           <property name="text">
            <string>BAJA</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="inTipo"/>
         </item>
         <item row="2" column="2">
          <widget class="QDateEdit" name="inFecha"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
